### ISSUE TITLE
Discussion Thread for `on-context.md`

### ISSUE THREAD

**Issue by dougli1sqrd** [rustbox/discussion#2]
Please provide thoughts on https://github.com/rustbox/discussion/blob/main/on-context.md here! :smile_cat:

**Comment by sethp** [comment]
Yes, to all of this. You've brilliantly captured some of the challenges (and pain) that I also feel. 

> But I have trouble translating these inspirations into a specific co-op idea.

Yeah, exactly: there's tremendous pressure to "do" "things" "fast", but will that produce a sustainable way for us to continue "doing" "things" (at _any_ pace?). I've got lots of ambitious ideas (as do you!), but they all feel like they've got odds more akin to powerball.

> Something I was thinking about recently then, was that this is what "business people" are for I guess, right?

Sorta; I read this paragraph as looking for someone we can ask for help with the more uncertain parts, right? Looking for prior expertise is a good instinct—we both know what it feels like when someone shows up and just kinda makes problems we thought were really hard go away (and/or says "ugh why did you do _that_?"). I think the tricky part is that "business people," broadly speaking, are evaluated on a single criteria: and "money number go up" isn't nearly as interesting to me as (for example) "wattage number go down." There's probably gems in so-called "management science" (what I'm given to understand is taught in Business School), but I don't think that an MBA would help us advance our goals. Or, at least, I don't think it's on the critical path.

What can we do if expert guidance isn't readily available? Well, just because we're not experts doesn't mean we can't try to solve it ourselves, right? We just have to trust each other (good news on that front), and maybe be a little overly optimistic about the future. Like:

> how to maybe get in touch with things people could need or want, [and] understand a path through the brambles from our visions and ideas into a concrete plan.

That feels like a workable problem to me. Plans, in my experience, aren't worth too much: but the work of generating a plan, of identifying areas of greatest uncertainty/risk and setting out to clarify those into a seemingly concrete series of steps, that's a skill worth practicing. It's hard to do, but it's something that benefits a lot from doing it together: I'll bet that I've got a lot of clarity on some of the "business-y" things that seem most fearsome to you, and I know you've helped me be less afraid of figuring out all the minuscule details before moving forward in the past.

> One "Plan" I thought of was that we could, in our free time, generate a series of "products" as kits that we could sell. And we'd go for breadth, generating a new gizmo every once in a while, basically building the kit per order. And then the hope would be that we'd have thrown enough technological spaghetti at the wall that something would stick and it'd take off. But that still feels sort of unsatisfying as it's just kinda guessing that something might work maybe someday.

I love this idea: to me, that sounds like a great way to practice the "planning" skill, and it adopts a semi-guided [Monte-Carlo search strategy](https://en.wikipedia.org/wiki/Monte_Carlo_tree_search) for the "does this fit a broad(er) need?" question. (Which, if you're curious, is how the Go game was "solved": just throw a lot of almost entirely loose spaghetti at the wall _really really fast_ across millions of dollars of computers. Easy, right?)

Another technique I've heard folks use for the search is to "scratch your own itch," because that makes getting immediate feedback trivial. That's what I'm going for with the router idea: I have a couple of network devices I'd like to replace, that fit a couple of different use-cases. From there, it's just a matter of us practicing getting through the brambles, right?
